samples/excel/89-preview-apis/workbook-get-active-cell.yaml

```diff
@@ -11,8 +11,8 @@ script:
         async function run() {
             await Excel.run(async (context) => {
 
-                const myWorkbook = context.workbook;
-                const activeCell = myWorkbook.getActiveCell();
+                let myWorkbook = context.workbook;
```

Comment:
I think `const `is TypeScript best practice when the value is not going to change. We haven't been consistent on this in the past, but I prefer `const`.